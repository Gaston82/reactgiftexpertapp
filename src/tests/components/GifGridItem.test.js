import React from "react";
import { shallow } from "enzyme";
import { GifGridItem } from "../../components/GifGridItem";

describe("Pruebas en <GifGridItem", () => {
  test("should show <GifGridItem/> correctly", () => {
    const wrapper = shallow(<GifGridItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
