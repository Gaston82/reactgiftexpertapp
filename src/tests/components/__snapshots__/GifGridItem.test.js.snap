// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem should show <GifGridItem/> correctly 1`] = `
<div
  className="card"
>
  <img />
  <p />
</div>
`;
